src/commands/workspace/start.ts

```diff
@@ -77,8 +77,8 @@ export default class Start extends Command {
       {
         title: 'Verify if Che server is running',
         task: async (ctx: any, task: any) => {
-          if (!await che.isCheServerReady(ctx.cheURL, flags.chenamespace)) {
-            this.error(`E_SRV_NOT_RUNNING - Che Server is not running.\nChe Server cannot be found in Kubernetes Namespace "${flags.chenamespace}". Have you already start it?\nFix with: start Che server: chectl server:start\nhttps://github.com/eclipse/che`, { code: 'E_SRV_NOT_RUNNNG' })
+          if (!await che.isCheServerReady(ctx.cheURL)) {
+            this.error(`E_SRV_NOT_RUNNING - Che Server is not running.\nChe Server is not available by ${ctx.cheURL}. T_O_`, { code: 'E_SRV_NOT_RUNNNG' })
```

Comment:
what's `T_O_`?

Reply:
Thanks for catching it.
It's TODO hidden from TODO bot that is actually disabled in this repo )
So, will revise this message and remove TODO

Reply:
TODO is removed and the message is changed to 
`E_SRV_NOT_RUNNING - Che Server is not available by ${ctx.cheURL}`